src/ios/UpdateHashUtils.m

```diff
@@ -68,18 +78,23 @@ + (NSString*)computeHashForData:(NSData*)inputData
     for (int i = 0; i < CC_SHA256_DIGEST_LENGTH; i++) {
```

Comment:
We should log manifest content here like this: log("Manifest string: " + manifestString);

Reply:
Are you sure the whole manifest should be logged like that? It can be quite huge, cluttering the logs.

Reply:
@afoo yeah, you are right, we should not do that.